constraint/Makefile

```diff
@@ -28,24 +28,29 @@ test: generate
 install: manifests
 	kubectl apply -f config/crds
 
+# Install the generation dependencies on the location machine
+gen-dependencies:
+	GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.5.0
+	GO111MODULE=on go get k8s.io/code-generator/cmd/conversion-gen@v0.21.0
```

Comment:
should we align this with controller-gen dependency version (v0.20.2)?

Reply:
We probably want to align this with the cert-controller dependency version (but that is also v0.20.2):

https://github.com/open-policy-agent/cert-controller/blob/c6f841d912f7d057a4e9fc3fe2c49951b68ba931/go.mod#L9-L11

Reply:
Will update